src/pytkdocs/objects.py

```diff
@@ -173,20 +173,26 @@ def relative_file_path(self) -> str:
 
         If the relative file path cannot be determined, the value returned is `""` (empty string).
         """
-        top_package_name = self.path.split(".", 1)[0]
-        try:
-            top_package = sys.modules[top_package_name]
-        except KeyError:
+
+        subspace_count = len(self.path.split("."))
+        namespaces = [".".join(self.path.split(".", maxsplit=maxsplit)[:-1]) for maxsplit in range(1, subspace_count)]
```

Comment:
So with `self.path == "a.b.c"`, `namespaces == ["a", "a.b"]`, right?

This would be simpler and more efficient I think:

```python
parts = self.path.split(".")
namespaces = [".".join(parts[:l]) for l in range(1, len(parts))]
```

Reply:
Nice!